crates/rpc-replay/Cargo.toml

```diff
@@ -16,6 +16,7 @@ starknet_api = { workspace = true }
 starknet-os-types = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true }
+serial_test = "3.2.0"
```

Comment:
This is a dev dependency. +1 if we can avoid adding it

Reply:
I have now updated it to be a dev-dependency, I hope that solves the issue,
This is required to ensure 
- `test_build_block_context_with_default_env_works`
- `test_build_block_context_with_default_env_fails`
works together properly.
WDYT ?